#ifndef MAINWINDOW_H
#define MAINWINDOW_H

#include <QMainWindow>
#include <QTcpSocket>
#include <QDebug>
#include <QTimer>

namespace Ui {
class MainWindow;
}

class MainWindow : public QMainWindow
{
  Q_OBJECT

public:
  explicit MainWindow(QWidget *parent = 0);
  ~MainWindow();
  
public slots:
  void tcpConnect();
  void tcpDisconnect();
  void putData();
  void startTimer();
  void stopTimer();
  void updateTimer();
private:
  Ui::MainWindow *ui;
  QTcpSocket *socket;
  QTimer *timer;
};

#endif // MAINWINDOW_H
